### PR TITLE
[deploy.sh] <PROJECT>_LOCAL_PORT for flask, express, spring-boot (instead of *_BACKEND_LOCAL)

### PR DESCRIPTION
**Pull request by realkosty**
## Scope
* [deploy.sh] <PROJECT>_LOCAL_PORT for flask, express, spring-boot (instead of *_BACKEND_LOCAL)
* Allow inline comments in *.env files

### Local
1. Edit env-config/local.env:
```
FLASK_LOCAL_PORT=8080
...
EXPRESS_LOCAL_PORT=8088
...
SPRINGBOOT_LOCAL_PORT=8090
```
2. Run:
```
./deploy.sh --env=local flask express spring-boot
```
3. Verify:
http://localhost:8080/products
http://localhost:8088/products
http://localhost:8090/products

### Cloud

1. Edit env-config/staging.env:
```
# NOTHING
```
2. Run:
```
./deploy.sh --env=staging flask express spring-boot
```
3. Verify:
https://staging-application-monitoring-flask-dot-sales-engineering-sf.appspot.com/products
https://staging-application-monitoring-node-dot-sales-engineering-sf.appspot.com/products
https://staging-springboot-dot-sales-engineering-sf.appspot.com/products

### CODE REVIEW

**Comment by Angelodaniel** [comment]
> Local
> Edit env-config/local.env:
> FLASK_LOCAL_PORT=8080
> ...
> EXPRESS_LOCAL_PORT=8088
> ...
> SPRINGBOOT_LOCAL_PORT=8090

This also means we can delete _APP_FLASK_BACKEND_LOCAL variables right? 
And we need RUBY_LOCAL_PORT=4567

File: env-config/example.env [hunk 1]
```diff
@@ -27,14 +23,17 @@ PASSWORD=
 CLOUD_SQL_CONNECTION_NAME=
 RUBY_BACKEND=
 FLASK_ENV=
+FLASK_LOCAL_PORT=8080 # only needed in local.env
 
 EXPRESS_APP_ENGINE_SERVICE=
 EXPRESS_SENTRY_PROJECT=
-EXPRESS_PORT=
+EXPRESS_LOCAL_PORT=
```
Comment (Angelodaniel) [inline env-config/example.env:30]: this looks like a duplicate

File: env-config/example.env [hunk 2]
```diff
@@ -27,14 +23,17 @@ PASSWORD=
...
 EXPRESS_ENV=
+EXPRESS_LOCAL_PORT=8088 # only needed in local.env
 
 RUBY_APP_ENGINE_SERVICE=
 RUBY_SENTRY_PROJECT=
+RUBY_LOCAL_PORT=4567 # only needed in local.env
```
Comment (Angelodaniel) [inline env-config/example.env:36]: Is there nothing we have to do with our current Ruby setup when changing this variable?

**Review by Angelodaniel — APPROVED** [review]

File: env-config/example.env (hunk 2 above)
Comment (realkosty) [reply env-config/example.env:36]: Good question. I didn't see any port specified anywhere in our Ruby code explicitly.
When I tried running it, looks like it's using that port by default:

```
./run.sh
[2022-12-19 16:10:47] INFO  WEBrick 1.4.4
[2022-12-19 16:10:47] INFO  ruby 2.6.10 (2022-04-12) [universal.arm64e-darwin21]
== Sinatra (v2.2.0) has taken the stage on 4567 for development with backup from WEBrick
[2022-12-19 16:10:47] INFO  WEBrick::HTTPServer#start: pid=14821 port=4567
```

**Pull request merged**